data/elevation-and-terrain/2016-lidar-kanab/index.md

```diff
@@ -0,0 +1,50 @@
+---
+status: publish
+layout: page
+tags:
+  - dem
+  - lidar
+  - elevation
+author:
+  display_name: Rick Kelson
+  email: rkelson@utah.gov
+published: true
+date: 2019-04-22 15:30:30
+title: 2016 Kanab Utah LiDAR Elevation Data
+categories: []
+---
+
+<style type="text/css">
+#logo {
+  max-width: 400px;
+  margin: 0 auto;
+}
+</style>
+<div id="logo">
+  <img src="{{ "/images/lidar_kanab_2016.png" | prepend: site.baseurl }}" alt="Sample" />
+</div>
+
+[![Project area map]({{"/images/lidar_kanab_2016_coverage_sm.png" | prepend:site.baseurl}} "click for map")]({{"/images/lidar_kanab_2016_coverage.png" | prepend:site.baseurl}}){:.inline-text-right}
+
+In 2016 the Natural Resource Conservation Service (NRCS) acquired LiDAR around Fredonia, AZ that covers [~150 square miles]({{ "/images/lidar_kanab_2016_coverage.png/" | prepend: site.baseurl }}) of Utah with 8 points per meter Quality Level 1 LiDAR and includes Kanab, UT. The .5 meter resolution bare earth DEMs and first-return/highest-hit DSMs in .img format have a 10.0cm vertical RMSE accuracy and are available for download. The LAS point clouds, classified to ground, low, medium, and high vegetation, are also available [by request](mailto:rkelson@utah.gov). This elevation data was collected between October and December, 2016 and has a UTM NAD83 (2011) zone 12 north meters NAVD88(GEOID12) projection.
```

Comment:
```suggestion
{% capture pcloud-dload %}{% include contact.html subject=page.title contact=site.data.contacts.elevation text='by request'%}{% endcapture %}

In 2016 the Natural Resource Conservation Service (NRCS) acquired LiDAR around Fredonia, AZ that covers [~150 square miles]({{ "/images/lidar_kanab_2016_coverage.png/" | prepend: site.baseurl }}) of Utah with 8 points per meter Quality Level 1 LiDAR and includes Kanab, UT. The .5 meter resolution bare earth DEMs and first-return/highest-hit DSMs in .img format have a 10.0cm vertical RMSE accuracy and are available for download. The LAS point clouds, classified to ground, low, medium, and high vegetation, are also available {{ pcloud-dload | strip_newlines }}. This elevation data was collected between October and December, 2016 and has a UTM NAD83 (2011) zone 12 north meters NAVD88(GEOID12) projection.
```

no mailto links. Use [contact information](https://github.com/agrc/gis.utah.gov#contact-information)

ps - my suggestion my not be formatted properly